activestorage/config/routes.rb

```diff
@@ -28,7 +28,7 @@
   resolve("ActiveStorage::Preview") { |preview, options| route_for(ActiveStorage.resolve_model_to_route, preview, options) }
 
   direct :rails_blob do |blob, options|
-    route_for(:rails_service_blob, blob.signed_id, blob.filename, options)
+    route_for(ActiveStorage.resolve_model_to_route, blob, options)
```

Comment:
If we are going to do this here, it probably makes sense to do the same sort of thing for `rails_representation` as well.

Reply:
I think, that make sense. Thanks for pointing it out!